src/Components/Atoms/Button/index.tsx

```diff
@@ -1,12 +1,12 @@
-import style from './ButtonPattern.module.scss'
+import style from './Button.module.scss'
 
 type Props = {
     type: "button" | "submit" | "reset" | undefined;
     onClick?: () => void;
     mainText?: string;
```

Comment:
Só pra manter um padrão q geral tende a usar, também deixa essa prop como obrigatoria
```suggestion
    label: string;
```